### PR TITLE
Default methods

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -11,12 +11,6 @@ end
 # ADNLPModel with no functions
 model = ADNLPModel(x->dot(x,x), zeros(2), name="square")
 @assert model.meta.name == "square"
-for meth in filter(f -> isa(eval(f), Function), names(NLPModels))
-  meth in (:reset!, :hess_op) && continue
-  meth in fieldnames(model.counters) && continue
-  meth = eval(meth)
-  @test_throws(NotImplementedError, meth(model))
-end
```
Comment (abelsiqueira) [inline test/runtests.jl]: Are you gonna with a DummyModel?

Comment (dpo) [inline test/runtests.jl]: Sorry I forgot.

Comment (dpo) [inline test/runtests.jl]: Done

**Comment by abelsiqueira** [comment]
Thanks, can you rebase it?

**Comment by dpo** [comment]
Rebased

**Comment by abelsiqueira** [comment]
Thanks

**Pull request merged**